addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.js

```diff
@@ -36,7 +36,7 @@ export class CalendarCommonPopover extends Component {
         } else {
             format = formattersRegistry.get(field.type);
         }
-        return format(record.data[fieldName]);
+        return field.type === 'html' ? '' : format(record.data[fieldName]);
```

Comment:
Could you do that in xml directly? i.e. not calling getFormattedValue if field type is html? Because here, the function could (in a custo for instance) be used for something else than the tooltip, and you won't get the value anymore for html fields.